src/main/java/frc/robot/commands/AutoScoreAndBalance.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.commands;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Calibrations;
+import frc.robot.subsystems.Lift;
+import frc.robot.subsystems.drive.DriveSubsystem;
+
+public class AutoScoreAndBalance extends SequentialCommandGroup {
+    private PathPlannerTrajectory traj = PathPlanner.loadPath("Trajectory",
+            new PathConstraints(Calibrations.PLACEHOLDER_DOUBLE, Calibrations.PLACEHOLDER_DOUBLE));
+    public AutoScoreAndBalance(boolean isFirstPath, Lift lift, DriveSubsystem drive) {
+        addCommands(
+                new InstantCommand(lift::scoreGamePiece, lift),
+                drive.followTrajectoryCommand(traj, isFirstPath),
+                new AutoChargeStationBalance(drive));
```

Comment:
We already have the charge station sequence, which drives up onto the charge station and balances. Either we should use that here (so have a path from scoring to *next to* the charge station), or scrap that sequence and just have every auton have a path to go from scoring/intaking to already on the charge station. We're never going to use the sequence in isolation because we won't have an auton that *only* balances but doesn't score or intake. @balghane thoughts?

Reply:
(This is for lines 19 and 20.)